addons/mrp/migrations/15.0.2.1/pre-migration.py

```diff
@@ -0,0 +1,8 @@
+def migrate(cr, version):
+    cr.execute(
+        """
+        UPDATE mrp_production as mp
+        SET date_start = (SELECT MIN(date_start) FROM mrp_workorder WHERE production_id = mp.id) 
+        WHERE mp.state != 'draft' AND mp.date_start IS NULL
```

Comment:
```suggestion
        UPDATE mrp_production as mp
           SET date_start = (SELECT MIN(date_start) FROM mrp_workorder WHERE production_id = mp.id)
         WHERE mp.state != 'draft' AND mp.date_start IS NULL
```
fix ci/style, improve readability

Reply:
@cawo-odoo ok, I edited the code, please review again

Reply:
Not exactly, what I meant. Please have a look at e.g. https://www.sqlstyle.guide/#do to get the idea and use a fixed-width font in your editor to get it right. Thank you.

Reply:
@cawo-odoo Ok, I got it

Reply:
@cawo-odoo Please review the code and let me know if any further modifications are needed. Thanks you :+1:

Reply:
Thanks for changing this. I think the upgrade is fine. The functional side must be reviewed by @odoo/rd-logistics , though

Reply:
@cawo-odoo Who can I tag to review this PR? My thousands of MO are facing the above situation.